<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purchasehistory</class>
 <widget class="QDialog" name="purchasehistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgb(238, 238, 238);

}



QPushButton
{
	border-radius: 12px;
	background-color: rgb(255, 211, 105);
	color: rgb(34, 40, 49);
	font: 700 11pt &quot;DejaVu Sans Mono&quot;;
	
}

QTableWidget
{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	padding: 5px;
	text-align: left;
	
}


QFrame
{

background-color: rgb(34, 40, 49);

}

QLabel
{
	background-color:rgb(238, 238, 238);
	color: rgb(34, 40, 49);
	font: 11pt &quot;DejaVu Sans Mono&quot;;
}

QLineEdit
{

	color:rgb(238, 238, 238);
	background-color: rgb(34, 40, 49);
	border-radius: 9px;
	font: 9pt &quot;DejaVu Sans Mono&quot;;

}
</string>
  </property>
  <widget class="QPushButton" name="displayHistory">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Ubuntu&quot;;
background-color: rgb(34, 40, 49);
color:rgb(238, 238, 238);

</string>
    </property>
    <property name="text">
     <string>Purchase History</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clearScreen">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>531</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
text-align: left;
font: 11pt &quot;DejaVu Sans Mono&quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>Product Code</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>154</red>
      <green>153</green>
      <blue>150</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>154</red>
      <green>153</green>
      <blue>150</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit Price</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>154</red>
      <green>153</green>
      <blue>150</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>154</red>
      <green>153</green>
      <blue>150</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>154</red>
      <green>153</green>
      <blue>150</blue>
     </color>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberIn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line2_2">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>170</y>
     <width>1300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(57, 62, 70);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorNumber">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>tableWidget</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>numberIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
